.github/workflows/ios-test.yml

```diff
@@ -12,6 +12,8 @@ jobs:
           node-version: 16
       - name: Lock xcode version
         run: sudo xcode-select -switch /Applications/Xcode_12.4.app
+      - name: Install global modules
+        run: npm ci --no-update-notifier --no-fund
```

Comment:
Uh, now that we need the global packages not just for `lint` and `test`, but also for `build`, `build.js` should _really_ include this part. Otherwise everybody needs to run this weird command line manually when they want to make a local build (but not when they want to trigger linting locally).

Reply:
You're right about that. Also, I will need to rebase to main.